Installation/cmake/modules/CGAL_GeneratorSpecificSettings.cmake

```diff
@@ -8,8 +8,11 @@ if ( NOT CGAL_GENERATOR_SPECIFIC_SETTINGS_FILE_INCLUDED )
     set(CGAL_AUTO_LINK_ENABLED TRUE)
   endif()
 
-  if     ( MSVC15 )
-    set(CGAL_TOOLSET "vc150")
+  if     ( MSVC16 )
+    set(CGAL_TOOLSET "vc142")
+    message( STATUS "Using VC16 compiler." )
+  elseif ( MSVC15 )
```

Comment:
`MSVC15` and `MSVC16` [do not exist], and never will. [`MSVC14`] and previous numbers have been deprecated. [`MSVC_VERSION`] (support since CMake-3.0 at least) is supposed to be used instead.

Since CMake 3.12, [`MSVC_TOOLSET_VERSION`] can be used directly to get the toolset. I think we could documented that recent MSVC versions require recent CMake versions. That is already true:
  - MSVC 2017 is supported since CMake 3.8,
  - MSVC 2019 is supported since CMake 3.14 only.

[`MSVC_TOOLSET_VERSION`]: https://cmake.org/cmake/help/v3.12/variable/MSVC_TOOLSET_VERSION.html

[do not exist]: https://cmake.org/cmake/help/v3.15/manual/cmake-variables.7.html#variables-that-describe-the-system
[`MSVC14`]: https://cmake.org/cmake/help/v3.15/variable/MSVC14.html
[`MSVC_VERSION`]: https://cmake.org/cmake/help/v3.15/variable/MSVC_VERSION.html

Reply:
It should be good now

Reply:
You can remove the test of MSVC15, because that variable does not exist.